src/Controller/AdminController.php

```diff
@@ -327,7 +327,10 @@ public function findAction(Request $request)
 
         //filter for query
         if (!empty($query)) {
-            $conditionParts[] = '(name LIKE ? OR description LIKE ? OR category LIKE ?)';
+            $conditionParts[] = sprintf('(%s LIKE ? OR %s LIKE ? OR category LIKE ?)',
+                $searcherList->quoteIdentifier('name'),
```

Comment:
this doesn't work with Pimcore 6.9 as `quoteIdenitifier` added to the Abstractlisting on 10.4. Need to call quoteIdentifier on db connection to make it work.

Reply:
ah, thank you for the hint, i was wondering why it was working on web2print but not there